ajax/libs/tiny-slider/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "tiny-slider",
+  "filename": "tiny-slider.js",
```

Comment:
Let's modify as `"filename": "min/tiny-slider.js",` 
Because we use minify file as the filename.
Thank you